### PR TITLE
Fix to improve WAVE accessibility score

### LINKED ISSUE #4082: Help us improve our WAVE accessibility score
https://wave.webaim.org/report#/openlibrary.org

### Evidence / Screenshot (if possible)

wave webaim org_report

### Relevant url?

https://wave.webaim.org/report#/openlibrary.org

### Proposal & Constraints

On homepage:
* [ ] 168 broken ARIA references
* [ ] 3 empty buttons
* [ ] 2 missing form labels

### Stakeholders

@bfalling

### PR DESCRIPTION
**Pull request by nynaalekhya**
Closes #4082 

### Technical

Improves WAVE accessibility score
### Screenshot

wave

### Stakeholders

@mekarpeles @bfalling

### CODE REVIEW

**Comment by nynaalekhya** [comment]
@mekarpeles @bfalling Please review this PR.

**Comment by mekarpeles** [comment]
Thank you @nynaalekhya for this contribution!

**Comment by nynaalekhya** [comment]
@jamesachamp Is there anything that I need to fix in this issue?

File: openlibrary/plugins/openlibrary/js/index.js [hunk 1]
```diff
@@ -139,6 +145,16 @@ jQuery(function () {
...
+    if ($('#cboxPrevious').length) {
+        $('#cboxPrevious').remove();
+    }
+    if ($('#cboxNext').length) {
+        $('#cboxNext').remove();
+    }
+    if ($('#cboxSlideshow').length) {
+        $('#cboxSlideshow').remove();
+    }
+
```
Comment (jamesachamp) [inline openlibrary/plugins/openlibrary/js/index.js:157]: I'm not really comfortable with removing these buttons.  These are added by Colorbox, the library that we use for our modals.  My worry is that removing these may inadvertently break something in one of our modals.

Comment (nynaalekhya) [reply openlibrary/plugins/openlibrary/js/index.js:157]: @jamesachamp 
Aria label attribute could be added to the buttons instead. This wouldn't require buttons to be removed.

Comment (nynaalekhya) [reply openlibrary/plugins/openlibrary/js/index.js:157]: And should I include the attribute aria hidden as well?

Comment (jamesachamp) [reply openlibrary/plugins/openlibrary/js/index.js]: I think that the changes you've made are good. Thanks so much!

**Review by jamesachamp — APPROVED** [review]

**Comment by nynaalekhya** [comment]
@jamesachamp Thank you

**Pull request merged**